src/components/dialogs/network-modifications/static-var-compensator/creation/susceptance-area.tsx

```diff
@@ -0,0 +1,177 @@
+
+    useEffect(() => {
+        let avgSfixeValue = (getFloatNumber(minS) + getFloatNumber(maxS)) / 2;
+        let avgQfixeValue = (getFloatNumber(minQ) + getFloatNumber(maxQ)) / 2;
+        setValue(`${id}.${B0}`, getFloatNumber(avgSfixeValue));
+        setValue(`${id}.${SLIDER_SUSCEPTANCE}`, getFloatNumber(avgSfixeValue));
+        setValue(`${id}.${Q0}`, getFloatNumber(avgQfixeValue));
+        setValue(`${id}.${SLIDER_Q_NOMINAL}`, getFloatNumber(avgQfixeValue));
+    }, [setValue, minS, minQ, maxS, maxQ, id]);
+
+    const onSliderSusceptanceChange = (value: string) => {
+        setValue(`${id}.${B0}`, getFloatNumber(value));
+        return value;
+    };
+
+    const onSliderQnomChange = (value: string) => {
+        setValue(`${id}.${Q0}`, getFloatNumber(value));
+        return value;
+    };
+
+    const handleSusceptanceValueChange = (value: string) => {
+        setValue(`${id}.${SLIDER_SUSCEPTANCE}`, getFloatNumber(value));
+        return value;
+    };
+
+    const handleQnomValueChange = (value: string) => {
+        setValue(`${id}.${SLIDER_Q_NOMINAL}`, getFloatNumber(value));
+        return value;
+    };
+
+    const minSusceptanceField = (
+        <TextField
+            value={minS}
+            label={<FormattedMessage id={'minSusceptance'} />}
+            disabled={true}
+            size={'small'}
+            InputProps={{
+                endAdornment: <InputAdornment position="start">S</InputAdornment>,
+            }}
+        />
+    );
+
+    const maxSusceptanceField = (
+        <TextField
+            value={maxS}
+            label={<FormattedMessage id={'maxSusceptance'} />}
+            disabled={true}
+            size={'small'}
+            InputProps={{
+                endAdornment: <InputAdornment position="start">S</InputAdornment>,
+            }}
+        />
+    );
+
+    const minQAtNominalVField = (
+        <TextField
+            value={minQ}
+            label={<FormattedMessage id={'minQ'} />}
+            disabled={true}
+            size={'small'}
+            InputProps={{
+                endAdornment: <InputAdornment position="start">MVA</InputAdornment>,
+            }}
+        />
+    );
+
+    const maxQAtNominalVField = (
+        <TextField
+            value={maxQ}
+            label={<FormattedMessage id={'maxQ'} />}
+            disabled={true}
+            size={'small'}
+            InputProps={{
+                endAdornment: <InputAdornment position="start">MVA</InputAdornment>,
+            }}
+        />
+    );
+
+    const susceptanceField = (
+        <TextInput
+            name={`${id}.${B0}`}
+            label="b0"
+            adornment={SusceptanceAdornment}
+            acceptValue={isValidPercentage}
+            outputTransform={handleSusceptanceValueChange}
+        />
+    );
+
+    const qAtNominalVField = (
+        <TextInput
+            name={`${id}.${Q0}`}
+            label="q0"
```

Comment:
![CSPR_Unit](https://github.com/user-attachments/assets/dc0b6dd2-dda1-4a1d-9dec-5cbe1f9a63e1)
unit shown here due to a shared translation key which contains unit
I have no solution for that except creating another translation key